jans-cedarling/cedarling/src/init/test.rs

```diff
@@ -0,0 +1,39 @@
+/*
+ * This software is available under the Apache-2.0 license.
+ * See https://www.apache.org/licenses/LICENSE-2.0.txt for full text.
+ *
+ * Copyright (c) 2024, Gluu, Inc.
+ */
+
+use super::policy_store::{load_policy_store, LoadPolicyStoreError};
+use crate::PolicyStoreConfig;
+
+/// Tests the behavior of the `load_policy_store` function when multiple role mappings
+/// are defined in the policy store configuration.
+///
+/// This test verifies that:
+/// 1. An error is returned when the policy store JSON contains multiple tokens with
+///    a `role_mapping` field.
+/// 2. The specific error returned is of type `LoadPolicyStoreError::MultipleRoleMappings`
+///    with the correct token kinds included in the error message.
+///
+/// The test uses a sample JSON file (`policy-store_with_multiple_role_mappings_err.json`)
+/// that is expected to trigger this error scenario.
+#[test]
+fn errors_on_multiple_role_mappings() {
```

Comment:
I would like move this unit test to file with `load_policy_store` function

Reply:
Also It would be great go add success case with check that all loaded with data what we need.

Reply:
Yep, that file is small. So it will be ok.

https://doc.rust-lang.org/rust-by-example/testing/unit_testing.html

Reply:
Or in file `policy_store_test.rs` near `policy_store.rs`

Reply:
okay, moved it here: [b600fdb](https://github.com/JanssenProject/jans/pull/9893/commits/b600fdba31e7ed053945541102f52df0a5487158)